ads/sovrn.js

```diff
@@ -16,19 +16,21 @@
  * Existing sovrn customers feel free to contact amp-implementations@sovrn.com for assistance with setting up your amp-ad tagid
  * New customers please see www.sovrn.com to sign up and get started!
  *********
+ **/
 /**
  * @param {!Window} global
  * @param {!Object} data
  */
-export function sovrn(global, data) {
+ import {writeScript} from '../src/3p';
+ export function sovrn(global, data) {
```

Comment:
It looks like the code indented an extra space from the left edge. Could you please shift it one space back?